docs/build-and-deploy-ground-from-source.md

```diff
@@ -275,13 +254,13 @@ stored on GitHub.
     };
     ```
 
-1. In a terminal window, from the _ground-platform_ directory:
+5. In a terminal window, from the _ground-platform_ directory:
     1. Run **`nvm install 14 && nvm use 14`**
```

Comment:
should this be nvm 16?

Reply:
Yes, though I didn't update these since these docs are not intended for developers. I will remove them for now to avoid confusion. Filed #1061 to track.